apps/prairielearn/src/ee/lib/aiQuestionGeneration.sql

```diff
@@ -28,3 +28,55 @@ FROM
 WHERE
   doc_path = $doc_path
   AND chunk_id = ANY ($chunk_ids);
+
+-- BLOCK insert_draft_question_metadata
+INSERT INTO
+  draft_question_metadata (question_id, created_by, updated_by)
+SELECT
+  $question_id,
+  u.user_id,
+  u.user_id
+FROM
+  users AS u
+WHERE
+  u.uid = $creator_id;
```

Comment:
What's going on here? Is `creator_id` a user ID or a UID? It looks like the former.

Reply:
changed in d830080b8b22c8440c018379442f76ca019db963